homeassistant/components/renson/sensor.py

```diff
@@ -63,29 +63,28 @@ class RensonSensorEntityDescriptionMixin:
 class RensonSensorEntityDescription(
     SensorEntityDescription, RensonSensorEntityDescriptionMixin
 ):
-    """Description of sensor."""
+    """Description of a Renson sensor."""
 
 
 SENSORS: tuple[RensonSensorEntityDescription, ...] = (
     RensonSensorEntityDescription(
         key="CO2_QUALITY_FIELD",
-        name="CO2 quality category",
+        translation_key="co2_quality_category",
         field=CO2_QUALITY_FIELD,
         raw_format=False,
         device_class=SensorDeviceClass.ENUM,
         options=["good", "bad", "poor"],
```

Comment:
Would it make sense to translate the states as well?

Reply:
We translate the good bad poor ones

Reply:
Ah ok, didn't know :)

Reply:
I mean it's in the strings.json with this PR :)

Reply:
Yea, indeed 🤦‍♂️